spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -7,18 +7,11 @@ This section goes into more detail about how you can work with Spring Cloud Stre
 such as creating and running stream applications.
 --
 
-=== Introducing Spring Cloud Stream
+== Introducing Spring Cloud Stream
 
-The Spring Cloud Stream project allows a user to develop and run messaging microservices using Spring Integration.
-Just add `@EnableBinding` and run your app as a Spring Boot app (single application context).
-Spring Cloud Stream applications connect to the physical broker through bindings, which link Spring Integration
-channels to physical broker destinations, for either input (consumer bindings) or output (producer bindings).
-The creation of the bindings, and therefore their broker-specific implementation is handled by a binder, which is
-another important abstraction of Spring Cloud Stream. Binders abstract out the broker-specific implementation details.
-In order to connect to a specific type of broker (e.g. Rabbit or Kafka) you just need to have the relevant binder
-implementation on the classpath.
+Spring Cloud Stream allows developing and running messaging microservices easily using Spring Integration, by just adding `@EnableBinding` and running your app as a Spring Boot application. Spring Cloud Stream applications connect to the physical broker through bindings, which link Spring Integration channels to physical broker destinations, or either input (consumer bindings) or output (producer bindings). The creation of the bindings, and therefore their broker-specific implementation is handled by a binder, which isanother important abstraction of Spring Cloud Stream. Binders abstract out the broker-specific implementation details. In order to connect to a specific type of broker (e.g. Rabbit or Kafka) you just need to have the relevant binder implementation on the classpath.
```

Comment:
The intro sentence feels a bit too long.  How about "Spring Cloud Stream is an event-driven microservice framework.  Spring Cloud Stream builds upon Spring Boot to create a devops friendly microservice applications and Spring Integration to provide connectivity to message brokers.  By adding `@EnableBinding` to your main application, you get immediate connectivity to a message broker and by adding `@StreamListener` to a method, you will receive events for stream processing.  Spring Cloud stream provides an opinionated configuration of message brokers, introducing the concepts of Persistent Pub/Sub semancis, consumer groups and partitions across several middleware vendors.